schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_entry.json

```diff
@@ -763,6 +763,133 @@
          "uniqueItems": true,
          "rcsb_nested_indexing": true
       },
+      "database_2": {
+         "type": "array",
+         "items": {
+            "type": "object",
+            "properties": {
```

Comment:
It seems that the `pdbx_database_accession` is missing

Reply:
Thanks for picking this up @valasatava. 

This has been addressed in the latest commit. Please review.